### PR TITLE
Apply patch generated by release tool

### PR DESCRIPTION
**Pull request by inteon**

### CODE REVIEW

File: config/jobs/cert-manager/cert-manager/master/cert-manager-master.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # THIS FILE HAS BEEN AUTOMATICALLY GENERATED
 # Don't manually edit it; instead edit the "cmrel" tool which generated it
-# Generated with: cmrel generate-prow --branch=master
+# Generated with: cmrel generate-prow --branch * -o file
```
Comment (irbekrm) [inline config/jobs/cert-manager/cert-manager/master/cert-manager-master.yaml:3]: How come this changed? Probably we want to keep `--branch=master` as that makes it obvious that this was generated for master. Same in other places

**Review by irbekrm — CHANGES_REQUESTED** [review]
Looks good, just added a nit about autogenerated file header change!

File: config/jobs/cert-manager/cert-manager/master/cert-manager-master.yaml (hunk 1 above)
Comment (inteon) [reply config/jobs/cert-manager/cert-manager/master/cert-manager-master.yaml:3]: @JoshVanL altered the script to generate all branches when providing the "*" value.
see https://github.com/cert-manager/release/pull/89

**Review by irbekrm — APPROVED** [review]
Thanks Tim!

/lgtm

**Pull request merged**